### PR TITLE
Add CMake build support under windows

### PR DESCRIPTION
**Pull request by zulkarnine**
Fix a common problem with QTDIR
Fix compile and link errors while building the project in windows using CMake and (VS2013)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,8 @@ project (DLT-Viewer)
 
 set(CMAKE_CXX_STANDARD 11)
 
+set(CMAKE_PREFIX_PATH $ENV{QTDIR})
```
Comment (tobsan) [inline CMakeLists.txt]: Shouldn't this be `set(CMAKE_PREFIX_PATH "${CMAKE_PREFIX_PATH} $ENV{QTDIR}")` instead, to avoid overwriting existing prefix path?

Comment (zulkarnine) [reply CMakeLists.txt]: Yes, that would be safer. I will change it

**Comment by GeniviDLTmaintainer** [comment]
Comes with next push from local muster ...

**Comment by zulkarnine** [comment]
> Comes with next push from local muster ...
What does it mean?

**Pull request closed without merging**